docs/03-tutorials/00-api-exposure/apix-03-own-domain.md

```diff
@@ -4,135 +4,48 @@ title: Use a custom domain to expose a service
-      name: cert-controller-manager
-      namespace: gardener-components
-    EOF
-    ```
+> **NOTE:** This tutorial uses the External DNS Management v.0.10.4 and the Certificate Management v0.8.3.
 
 ## Steps
 
 Follow these steps to set up your custom domain and prepare a certificate required to expose a service.
 
-1. [Install Kyma](../../04-operation-guides/operations/02-install-kyma.md) 2.0 or higher.
-
-2. Create a Namespace. Run:
+1. Create a Namespace. Run:
 
-   ```bash
+  ```bash
    kubectl create ns {NAMESPACE_NAME}
    ```
 
-3. Create a Secret containing credentials for your DNS cloud service provider account.
+2. Create a Secret containing credentials for your DNS cloud service provider account in your Namespace.
 
-- See the [official External DNS Management documentation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines.
-- Remember to edit the **metadata.namespace** parameter and provide your {NAMESPACE_NAME} as the required value.  
-- Once the YAML file with the relevant parameters is ready, create a Secret Custom Resource (CR). Run the following command:
+  See the [official External DNS Management documentation](https://github.com/gardener/external-dns-management/blob/master/README.md#external-dns-management), choose your DNS cloud service provider, and follow the relevant guidelines. Then run:
 
-   ```bash
-   kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
-   ```
+  ```bash
+  kubectl apply -n {NAMESPACE_NAME} -f {SECRET}.yaml
+  ```
 
-4. Set up the `external-dns-management` component.
+3. Create a DNSProvider and a DNEntry CRs.
 
-- Create a DNSProvider CR. Export values of the **metadata.namespace**, [**spec.type**](https://github.com/gardener/external-dns-management#using-the-dns-controller-manager), **metadata.spec.secretRef.name** and **metadata.spec.domains.include** parameters as environment variables.
-- As the value of **spec.type**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR. 
-- For the **spec.secretRef.name** parameter, use the **metadata.name** value from your `{SECRET}.yaml`.
-- The domain you provide is the one that you own, for example `mydomain.com`. In the next steps you provide a subdomain of this domain, for example `api.mydomain.com`. Run:
+> **CAUTION:** Bear in mind that the **metadata.annotation** parameter, may be either not needed or subject to change depending on the External DNS Management configuration provided during the component installation.
+
+- Export the following values as environment variables and run the command provided. 
+  
+  As the **SPEC_TYPE**, use the relevant provider type. See the [official Gardener examples](https://github.com/gardener/external-dns-management/tree/master/examples) of the DNSProvider CR.
```

Comment:
I think we need 3 (?) additional blank spaces for proper indentation here. Also for the following sample code.